docs/api/navigators/DrawerNavigator.md

```diff
@@ -128,6 +128,7 @@ const styles = StyleSheet.create({
 
 ### `contentOptions` for `DrawerView.Items`
 
+- `routes` - an array of routes to use in place of `navigation.state.routes`
```

Comment:
What would you suggest? The original code called them routes everywhere, so I called the prop `routes`.